app/src/main/java/com/kickstarter/viewmodels/ActivityFeedViewModel.java

```diff
@@ -115,15 +114,15 @@ public ViewModel(final @NonNull Environment environment) {
           this.currentUser.isLoggedIn(),
           Pair::create
         )
-        .filter(resumedAndLoggedIn ->
-          resumedAndLoggedIn.first == ActivityEvent.RESUME && resumedAndLoggedIn.second
-        )
+        .map(resumedAndLoggedIn -> resumedAndLoggedIn.second)
```

Comment:
if you wanted to try for a stretch goal, you could create a lil pair util function so that we can do `map(second)`. we do this kind of thing quite often in swift and it's very convenient.

this is also an example of [point-free programming](https://en.wikipedia.org/wiki/Tacit_programming), which we are big into here.